modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -0,0 +1,141 @@
+
+. Create a directory on your local system named `openshift`. 
++
+[IMPORTANT]
+====
+Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
+====
+
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, select the **<> Code** button and click **Download ZIP**. Save the archive file to your `openshift` directory,  so that all the CCM and CSI manifests exist in the same directory. The downloaded archive file includes files for creating cluster resources and custom manifests. 
+
+. Go to link:https://github.com/oracle-quickstart/oci-openshift/tree/main/custom_manifests[CCM and CSI custom manifests] link to access the custom manifest files.
++
+The CCM manifest are required for deploying Oracle’s CCM during cluster installation so that {product-title} can connect to the external {oci} platform. The CSI custom manifests are required for deploying Oracle’s CSI driver during cluster installation so that {product-title} can claim required objects from {oci}.
++
+[IMPORTANT]
+====
+You must modify the secret `oci-cloud-controller-manager` defined in the link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/manifests/oci-ccm.yml[`oci-ccm.yml`] configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
+====
+
+. Use the Agent-based Installer to generate a minimal ISO image, which excludes the `rootfs` image, by entering the following command in your {product-title} CLI. You can use this image later in the process to boot all your cluster’s nodes. 
++
+[source,terminal]
+----
+$ ./openshift-install agent create image --log-level debug
+----
++
+The previous command also completes the following actions:
++
+* Creates a subdirectory, `./<installation_directory>/auth directory:`, and places `kubeadmin-password` and `kubeconfig` files in the subdirectory.  
+* Creates a `rendezvousIP` file based on the IP address that you specified in the `agent-config.yaml` configuration file. 
+* Optional: Any modifications you made to `agent-config.yaml` and `install-config.yaml` configuration files get imported to the Zero Touch Provisioning (ZTP) custom resources. 
++
+[IMPORTANT]
+====
+The Agent-based Installer uses {op-system-first}. The `rootfs` image, which is mentioned in a subsequent listed item,  is required for booting, recovering, and repairing your operating system. 
+====
+
+. Configure the `agent-config.yaml` configuration file to meet your organization’s requirements. 
++
+.Example `agent-config.yaml` configuration file that sets values for an IPv4 formatted network.
+[source,yaml]
+----
+apiVersion: v1alpha1
+metadata:
+  name: <cluster_name> <1>
+  namespace: <cluster_namespace> <2>
```

Comment:
No need to specify the metadata section, as it will be ignored by the installer, and the metadata from the install-config file will be used instead.

Reply:
Removed this segment from the config example.